src/test/java/tests/AddToCartTest.java

```diff
@@ -14,7 +15,9 @@ public void testAddProductToCart() {
         LoginPage loginPage = new LoginPage(getDriver());
         loginPage.open();
 
-        InventoryPage inventoryPage = loginPage.login("standard_user", "secret_sauce");
+        InventoryPage inventoryPage = loginPage.login(UserCredentials.STANDARD_USER.getUsername(),
```

Comment:
If you're using something like UserCredentials.STANDARD_USER.getUsername() in multiple places, it's a good idea to encapsulate login logic inside a UserService (or LoginService) for clarity, reuse, and maintainability.